modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/ScriptProcessor.java

```diff
@@ -53,7 +52,7 @@
 public final class ScriptProcessor extends AbstractProcessor {
 
     private static final DeprecationLogger deprecationLogger =
-            new DeprecationLogger(LogManager.getLogger(DynamicMap.class));
+            DeprecationLogger.getLogger(DynamicMap.class);
```

Comment:
there is a series of classes that possibly incorrectly use DynamicMap logger
This PR grew so large and was mostly automatic (find/replace, backport, automated refactoring) that I am reluctant to fix it here too. Maybe I miss something and `DynamicMap` was actually intended here?
@pugnascotia WDYT?

Reply:
Looks like a mistake to me. @stu-elastic what do you think?